client/src/pages/Login/Login.jsx

```diff
@@ -79,9 +79,17 @@ const Login = () => {
       }
       const result = await signInCall(values);
 
-      localStorage.setItem('user', JSON.stringify(result));
-      reset();
-      history.push('/home');
+      if (result === undefined) {
+        setError({
+          type: 'password',
+          description: 'Wrong Details',
+        });
+      } else {
+        localStorage.setItem('user', JSON.stringify(result));
+        reset();
+        history.push('/home');
+        window.location.reload();
```

Comment:
Why reload?

Reply:
strange I delete this in previous branches 😅😅. will delete it again